SimG4Core/PhysicsLists/interface/CMSParticleList.h

```diff
@@ -0,0 +1,25 @@
+#ifndef SimG4Core_PhysicsList_CMSParticleList_h
+#define SimG4Core_PhysicsList_CMSParticleList_h 1
+
+// V.Ivanchenko 6 March 2017
+// List of Geant4 basic particle names used in SIM step 
+
+#include "globals.hh"
+
+static const G4int numParticles = 44;
+static const G4String PartNames[numParticles] = 
```

Comment:
Hi @civanch - two changes needed here 
   -first move this into a namespace or class
   -second, declare PartNames as an extern const so that the strings themselves can go into a c++